src/commands/gist.ts

```diff
@@ -118,6 +122,56 @@ async function newGistInternal(isPublic: boolean = true, description: string = "
   descriptionInputBox.show();
 }
 
+async function syncGistInternal() {
+  await ensureAuthenticated();
+  
+  let error: Error | undefined;
+  const activeEditor = window.activeTextEditor;
+  if (!activeEditor) {
+    throw new Error("No active editor");
+  }
+  
+  await window.withProgress(
+    {
+      location: ProgressLocation.Notification,
+      title: "Uploading to GitHub Gist..."
+    },
+    async () => {
+      try {
+        const uri = activeEditor.document.uri;
+        const { gistId } = getGistDetailsFromUri(uri);
+        
+        if (!isOwnedGist(gistId)) {
```

Comment:
I suppose this could trigger a fork flow, but that's pretty low priority and could be worried about later if it turns out people actually run into it.